desktop/core/src/desktop/js/parse/jison/sql/hive/sql_main.jison

```diff
@@ -118,6 +119,7 @@ NonReservedKeyword
  | 'DATABASES'
  | 'DAY'
  | 'DAYOFWEEK'
+ | 'DCPROPERTIES'
```

Comment:
are we following alphabetical sorting? If yes then, 'DCPROPERTIES' should come after 'DBPROPERTIES'.

Reply:
Ah yes, in this case it doesn't matter but from a clean code perspective it's nice to have them sorted.

It could be good to know that when it comes to the suggestions the autocomplete makes it's not always alphabetically sorted, if a keyword has to come before another it's placed higher up. For instance in a SELECT statement the WHERE clause comes before the LIMIT clause and thus higher up as the order matters.

Reply:
Yes, Got it.
Thanks